SPECS/buildah/buildah.spec

```diff
@@ -0,0 +1,1973 @@
+%global with_bundled 1
```

Comment:
Seems unused. If that's correct, let's remove this macro.

Reply:
Right! It has been removed.